week3-task/src/components/GithubSearch/RecentSearches.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import {
+  recentSearchContainer,
+  recentSearchTitle,
+  recentSearchList,
+  recentSearchItem,
+  deleteButton,
+} from "../../styles/GithubSearch.style";
+
+const RecentSearches = ({ searches, onClick, onDelete, theme }) =>
+  searches.length > 0 && (
+    <div css={recentSearchContainer}>
+      <div css={recentSearchTitle}>최근 검색어</div>
+      <div css={recentSearchList}>
+        {searches.map((search) => (
+          <div
+            key={search}
+            css={recentSearchItem(theme)}
+            tabIndex={0}
+            onClick={() => onClick(search)}
+            onKeyDown={(e) => {
+              if (e.key === "Enter" || e.key === " ") onClick(search);
+            }}
+          >

```

Comment:
현재는 모두 `div`로 구성되어 있는데, 시맨틱 태그를 고려해 `ul` / `li` 구조로 변경하면 더 의미가 명확해질 것 같아요!
```suggestion
      <ul css={recentSearchList}>
        {searches.map((search) => (
          <li
            key={search}
            css={recentSearchItem(theme)}
            tabIndex={0}
            onClick={() => onClick(search)}
            onKeyDown={(e) => {
              if (e.key === "Enter" || e.key === " ") onClick(search);
            }}
          >

```